instrumentation/testing/tb.go

```diff
@@ -216,14 +218,19 @@ func (test *Test) Helper() {
 	test.t.Helper()
 }
 
+// Log panic data with stacktrace
+func (test *Test) LogPanic(recoverData interface{}, skipFrames int) {
```

Comment:
Is this function called automatically by test instrumentation, right?

Reply:
The internal test instrumentation uses `WriteExceptionEvent` because already have the `opentracing.Span` reference. `LogPanic` it's a very similar function but using the `context` to extract the span. Useful for others and manual instrumentation.

Reply:
This `LogPanic` method is exposed in `tb.go` too. So we can use:

```go
func TestBasic(t *testing.T) {
    test := scopeagent.GetTest(t)

    defer func() {
        if r := recover(); r != nil {
            test.LogPanic(r, 1)
        }
    }()

    panic("test")
}
```